azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/ResourceGroup.java

```diff
@@ -1,7 +1,7 @@
 package com.microsoft.azure.management.resources;
 
 import com.microsoft.azure.management.resources.fluentcore.arm.Region;
-import com.microsoft.azure.management.resources.fluentcore.arm.models.Taggable;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.Resource;
 import com.microsoft.azure.management.resources.fluentcore.model.*;
```

Comment:
Yes. This can probably be found in some sort of settings. In IntelliJ it's a threshold setting of how many classes in one package to import. Setting it to 99 instead of 6 gets the problem solved.